vaadin-combo-box-flow/src/main/resources/META-INF/resources/frontend/comboBoxConnector.js

```diff
@@ -168,10 +190,7 @@ window.Vaadin.Flow.comboBoxConnector = {
         let page = outstandingRequests[i];
 
         if (cache[page]) {
-          let callback = pageCallbacks[page];
-          delete pageCallbacks[page];
```

Comment:
Should no longer delete values from the `pageCallbacks` object as it represents the currently active range.